test_data/asm/batching/labels_batched.asm

```diff
@@ -1,21 +1,16 @@
-degree 256;
-reg pc[@pc];
-reg X[<=];
-reg A;
+program {
```

Comment:
Maybe add a comment saying this isn't valid assembly, and/or change the file extension to make it explicitly that this is a specific test for batching.